src/Microsoft.Health.Dicom.Api/DicomApiResource.resx

```diff
@@ -173,7 +173,7 @@
     <comment>{0} is field.</comment>
   </data>
   <data name="WarningHeader" xml:space="preserve">
-    <value>299: {0}</value>
-    <comment>{0} is the warning message</comment>
+    <value>299 {0}: "{1}"</value>
+    <comment>{0} is host name, {1} is the warning message</comment>
```

Comment:
This probably shouldn't be in here, as this `string` is not culture-specific.

Reply:
While no longer in use, this change still needs to be deleted

Reply:
yeah, forgot to do that...